### ISSUE TITLE
Make certificate validation optional

### ISSUE THREAD

**Issue by mountaindude** [ptarmiganlabs/butler-sos#178]
**Describe the bug**
Currently, when Butler SOS connects to QSEoW it doesn't reject self-signed certificates (which QSEoW uses by design).
This isn't really a problem in well secured internal networks, but nonetheless something that should be configurable. 

**Expected behavior**
Add an option in the Butler SOS config file to give the admin the choice of reject or accept self-signed certificates.

**Comment by mountaindude** [comment]
Config file setting added: `Butler-SOS.serversToMonitor.rejectUnauthorized'

When set to true, the following error will appear when getting metrics from a Sense host, using a host name other than what's embedded in Sense's self-signed certificate for that host:

```
2021-08-31T16:46:24.415Z error: HEALTH: Error when calling health check API: Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: pro2-win2.ptarmiganlabs.net. is not in the cert's altnames: DNS:pro2-win2.lab.ptarmiganlabs.net
```

This means that the TLS cert used to secure the Sense API was created for a computer named `pro2-win2.lab.ptarmiganlabs.net`, but the call to the API used `pro2-win2.ptarmiganlabs.net`..

Setting `rejectUnauthorized: false` will make Butler SOS disregard the cert mismatch.

The proper, most secure solution is however to use the correct host name in the config file, and keep `rejectUnauthorized: true`.

**Comment by mountaindude** [comment]
Released as part of [v6.0.0](https://github.com/ptarmiganlabs/butler-sos/releases/tag/v6.0.0)
